docs/common/guides/jlink_image.adoc

```diff
@@ -1,6 +1,6 @@
 ///////////////////////////////////////////////////////////////////////////////
 
-    Copyright (c) 2020 Oracle and/or its affiliates.
```

Comment:
We generally don't _replace_ the original year in the copyright notice; rather, we _add_ the new year using "Copyright (c) 2020, 2021" in a case like this. (I won't tag each occurrence of this in this PR.)